cinemas_repertoire/rest_views.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import viewsets
+from rest_framework.response import Response
+
+from .serializers import FilmSerializer
+from .utils import get_cinemas, get_movies, get_movie_details
+
+
+class EventsViewset(viewsets.ViewSet):
+
+    def list(self, request):
+        movie_data = [
+            {
+                "cinema": cinema['id'],
```

Comment:
You can also use a serializers to parse the data that comes from cinema city, e.g. 
```
CinemaCityEventSerializer(Serializer):
    film = CinemaCityFilmSerializer(...)
    
CinemaCityFilmSerializer(Serializer):
   length = ...
   release_year= ...
```
etc.